components/service-binding-usage-controller/docs/architecture.md

```diff
@@ -1,20 +1,20 @@
 
-2. The ServiceBindingUsage Controller fetches all labels from the **bindingLabels** section in the ClusterServiceClass for which the ServiceBinding was created.
+2. The Service Binding Usage Controller fetches all labels from the **bindingLabels** section in the Cluster Service Class for which the Service Binding was created.
 
-3. The ServiceBindingUsage Controller finds the corresponding UsageKind specified in the **usedBy.Kind** field. The UsageKind defines the resource to be labeled and the field which contains labels.
+3. The Service Binding Usage Controller finds the corresponding Usage Kind specified in the **usedBy.Kind** field. The Usage Kind defines the resource to be labeled and the field which contains labels.
 
-4. The ServiceBindingUsage Controller creates a PodPreset which is applied to all Pods with the **use-{serviceBindingUsageUID}** label set to `{serviceBindingUsage resource version}`. The name of the PodPreset is the SHA1-encoded name of the ServiceBindingUsage. The PodPreset injects a Secret with the same name as specified in the  **serviceBindingRef.name** field into those Pods. By default, the prefixing of the injected environment variables is disabled. Set **envPrefix.name** to enable the prefix. **envPrefix.name** is used to prefix the name of the environment variable.
+4. The Service Binding Usage Controller creates a Pod Preset which is applied to all Pods with the **use-{serviceBindingUsageUID}** label set to `{serviceBindingUsage resource version}`. The name of the Pod Preset is the SHA1-encoded name of the Service Binding Usage. The Pod Preset injects a Secret with the same name as specified in the  **serviceBindingRef.name** field into those Pods. By default, the prefixing of the injected environment variables is disabled. Set **envPrefix.name** to enable the prefix. **envPrefix.name** is used to prefix the name of the environment variable.
 
-5. The ServiceBindingUsage Controller adds the labels from steps 2 and 3 to the resource specified in the **usedBy** property.
+5. The Service Binding Usage Controller adds the labels from steps 2 and 3 to the resource specified in the **usedBy** property.
 
-6. The ServiceBindingUsage Controller adds an annotation with data on the applied ServiceBindingUsage to the Deployment. Annotations are used for tracing and debugging purposes. With annotations, you can trace the labels attached to a given Deployment. The pattern of such annotation is as follows:
+6. The Service Binding Usage Controller adds an annotation with data on the applied Service Binding Usage to the Deployment. Annotations are used for tracing and debugging purposes. With annotations, you can trace the labels attached to a given Deployment. The pattern of such annotation is as follows:
 
     ```
      servicebindingusages.servicecatalog.kyma-project.io/tracing-information: '{
```

Comment:
I'd change it in the line below too (unless you don't use spaces in the placeholder at all, but then we usually use `_`). 
```
"{Service Binding Usage name}": {
```